version/version.go

```diff
@@ -14,7 +14,7 @@ package version
 
 var (
 	// Version is the operator version
-	Version = "v0.6.0"
+	Version = "0.8.0-SNAPSHOT"
```

Comment:
I don't know the standard practice for go projects here, but `-SNAPSHOT` comes from maven conventions IIRC. 

I wonder if we should leave `Version` blank in non-release versions, and refer to builds by commit hash instead. On release, we could set this field appropriately.

Reply:
It's not a strong opinion but I would like to have someplace to understand which version is going to be next.
And if it's deployed somewhere, understand for which dirty version it belongs to without analyzing commits.

But yeah, I did not manage to find any other go project that stores the next version.
But AFAIS we should be able to append metainfo with +, like
```suggestion
	Version = "v0.8.0+draft"
```

Reply:
changed to what we had previously -> `v0.8.0` going to merge and we'll discuss what we'll do further.